.bcr/config.yml

```diff
@@ -12,7 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+---
 fixedReleaser:
-  login: f0rmiga
```

Comment:
I think leave this part as-is for now? I think the BCR app has to be installed for/by the user and authorized by the bazel org for that user and repo. Unless you've set that up already (seems like you'd have mentioned it before hand), lets leave this part be.